chainer/backends/intel64.py

```diff
@@ -10,9 +10,10 @@
 try:
     import ideep4py as ideep  # NOQA
     from ideep4py import mdarray  # NOQA
-    _ideep_version = 0
+    _ideep_version = 2 if hasattr(ideep, '__version__') else 1
```

Comment:
Why do you extract version number from `ideep.__version__` ?

Reply:
Because iDeep 1.x does not have `__version__`.

Reply:
iDeep 3 will have `__version__`.

Reply:
As discussed, I'll fix forr iDeep 3.x in separate PR.